src/main/java/org/gridsuite/study/server/networkmodificationtree/entities/NetworkModificationNodeInfoEntity.java

```diff
@@ -35,6 +35,13 @@ public class NetworkModificationNodeInfoEntity extends AbstractNodeInfoEntity {
     @Builder.Default
     protected List<RootNetworkNodeInfoEntity> rootNetworkNodeInfos = new ArrayList<>();
 
+    @ElementCollection
+    @CollectionTable(
+        name = "node_aliases",
+        joinColumns = @JoinColumn(name = "id_node", foreignKey = @ForeignKey(name = "node_aliases_node_id_node_fk_1"))
+    )
+    protected List<NodeAliasEmbeddable> nodeAliases = new ArrayList<>();
```

Comment:
Do we allow to define different aliases for the same node ?
If yes, any of these aliases could be used in a spreadsheet colum formula to identify the node, which is not the case : only the first alias can be used ...

Reply:
As discussed with PO, will be the subject of another issue